### PR TITLE
Don't deploy example apps in pull requests

### PR DESCRIPTION
**Pull request by gadenbuie**
Instead only deploy the apps on the push events to the `main` branch in rstudio/flexdashboard.

### CODE REVIEW

**Review by schloerke — CHANGES_REQUESTED** [review]
Can we do the `if` logic on line 26? Then everything is at the highest level

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -57,5 +60,5 @@ jobs:
             secret = '${{ secrets.SHINYAPPS_SECRET }}'
           )
       - name: Deploy
-        run: |
+        if: ${{ steps.examples.deploy == 'true' }}        run: |
```
Comment (schloerke) [inline .github/workflows/R-CMD-check.yaml]: Undo this change por favor

**Review by schloerke — APPROVED** [review]

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (gadenbuie) [reply .github/workflows/R-CMD-check.yaml]: lol I'm too fast 6fccc70be96f39203fbffba0db4b173b7a5d4064
